java/src/main/java/ai/rapids/cudf/HostColumnVectorCore.java

```diff
@@ -121,7 +121,7 @@ public long getNullCount() {
   /**
    * Returns the list of child host column vectors for a given host side column
    */
-  List<HostColumnVectorCore> getNestedChildren() {
+  public List<HostColumnVectorCore> getNestedChildren() {
     return children;
```

Comment:
Question- why do we need this change? I thought we did not want to expose this as getting to a child was available thru ColumnViewAccess APIs.

Reply:
You are right.  I did it while prototyping changes to the plugin.  I'll fix it.